psi4/src/core.cc

```diff
@@ -968,77 +887,52 @@ py::object py_psi_get_option(std::string const& module, std::string const& key)
     return py::object();
 }
 
-void py_psi_set_active_molecule(std::shared_ptr<Molecule> molecule)
-{
-    Process::environment.set_molecule(molecule);
-}
-void py_psi_set_legacy_molecule(std::shared_ptr<Molecule> legacy_molecule)
-{
+void py_psi_set_active_molecule(std::shared_ptr<Molecule> molecule) { Process::environment.set_molecule(molecule); }
+void py_psi_set_legacy_molecule(std::shared_ptr<Molecule> legacy_molecule) {
```

Comment:
No reason to do it immediately (that is, this PR), but many of these should just become lambdas in the pb11 `.def`.